### DISCUSSION TITLE
fr_dep_news_trf model on MacOS ARM

### DISCUSSION THREAD

**Discussion by dandiep** [explosion/spaCy#9177] [Help: Installation] [answered]
Hi, I'm trying to use the fr_dep_news_trf model on my ARM based Mac. The requirement seems to be that sentencepiece 0.1.91 is used, but that doesn't have binaries for Mac, nor does the source build seem to work on mac.

Is there any way to make the model work with newer sentencepiece versions that are ARM compatible? Thanks!

**Comment by polm** [comment, marked as answer]
It looks like the sentencepiece version is [hardcoded in Transformers](https://github.com/huggingface/transformers/blob/c37573806ab3526dd805c49cbe2489ad4d68a9d7/setup.py#L137), so there's not much we can do about this on our end. If you use a more recent version of sentencepiece it might just work?

**Comment by dandiep** [comment]
Ah, thanks, I didn't piece together it was Transformers itself that required this (no pun intended :-P). I will ask them about and experiment with newer versions.

When I try `python -m spacy download fr_dep_news_trf` it fails because the right version isn't installed. Is there a way to override this and force installation? I am somewhat new to Python so I apologize if there is something basic about dependency management here that I am missing.

**Reply by polm** [reply]
You can do this:

    python -m spacy download fr_dep_news_trf --no-deps

Extra arguments after the model name will go directly to `pip`, and `--no-deps` tells it to ignore requirements.

**Reply by dandiep** [reply]
Perfect. Thanks.
